frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding.unit.spec.tsx

```diff
@@ -115,5 +115,11 @@ describe("SettingsEditor", () => {
 });
 
 const goToInteractiveEmbeddingSettings = async () => {
-  await userEvent.click(screen.getByText("Configure"));
+  await userEvent.click(
+    within(
+      screen.getByRole("article", {
+        name: "Interactive embedding",
+      }),
+    ).getByText("Configure"),
```

Comment:
We now have 2 `Configure` buttons, so we need a way to target each individual embedding card.
![image](https://github.com/user-attachments/assets/5d147aef-9135-4afe-8a45-2d36d30bfe19)